src/lib/repositories_v2/impl/chef_repositoty_impl.py

```diff
@@ -0,0 +1,62 @@
+from datetime import datetime
+
+from src.constants.audit import Status
+from src.core.ioc import get_ioc_instance
+from src.lib.entities.sqlalchemy_orm_mapping import Chef, Order
+from src.lib.repositories_v2.chef_repository import ChefRepository
+
+
+class ChefRepositoryImpl(ChefRepository):
+    def __init__(self):
+
+        ioc = get_ioc_instance()
+        self.session = ioc.get_instance("orm_session")
+
+    def add(self, chef):
+        chef.created_date = datetime.now()
+        chef.updated_by = chef.created_by
+        chef.updated_date = chef.created_date
+        self.session.add(chef)
+        self.session.commit()
+
+    def get_by_id(self, chef_id):
+        return (
+            self.session.query(Chef)
+            .filter(Chef.id == chef_id)
+            .filter(Chef.entity_status == "ACTIVE")
```

Comment:
I think it is better to use the enum here instead of string, isn't it?

Reply:
done.